### ISSUE TITLE
wpcli `migrate` subcommand disappears after one use, help disappears, and not like `wp wc update`

### ISSUE THREAD

**Issue by diablodale** [woocommerce/action-scheduler#574]
`migrate` subcommand disappears after one use, its help disappears, and it behaves differently than commands like `wp wc update`.

https://github.com/woocommerce/action-scheduler/blob/f3a13eb9b5c9b1f06db1144caa219202c27deef6/classes/abstracts/ActionScheduler.php#L169-L172

## Setup

* Woo 4.2.0 which includes Ac 3.1.6
* Also reproduces on current Ac master as linked above

## Repo

1. Install a complete Woo setup by command-line (not UI or webpages)
2. `wpcli action-scheduler migrate`
3. `wpcli action-scheduler migrate`

## Result

Error and a non-zero exit code

```
Error: 'migrate' is not a registered subcommand of 'action-scheduler'. See 'wp help action-scheduler' for available subcommands.`
```

Further...when you follow those instructions to get help with `wp help action-scheduler` all help on it has vanished.

```
...
SUBCOMMANDS

  run      Run the Action Scheduler

GLOBAL PARAMETERS
...
```

## Expected

No error and action scheduler should provide a friendly notice that no migrate/update is needed and note the current version. For example, this is the friendly notice (with exit code=0) that Woo provides when `wp wc update` is run:

```
Success: No updates required. Database version is 4.2.0
```

Also, complete help should always be available. When `wp help action-scheduler` is run, it should provide a list of all subcommands (including `migrate`) and help for them all.

**Comment by rrennick** [comment]
> migrate subcommand disappears after one use,

@diablodale Thanks for the suggestion. Once to migration is complete the migrate command is no longer registered. I saw another mention of this elsewhere.

**Comment by rrennick** [comment]
Addressed in #575
